<launch>
    <arg name="dir"     value="/tmp"/>
    <arg name="left_topic"   value="/hcfox/image"/>
    <arg name="right_topic"  value="/hcfox/image"/>
    <node pkg="calib_image_saver" type="stereoImageSaver" name="saver" output="screen">
        <remap from="/left_image"   to="$(arg left_topic)"/>
        <remap from="/right_image"  to="$(arg right_topic)"/>
        <param name="image_path"        type="string" value="$(arg dir)"/>
        <param name="image_name_left"   type="string" value="left_"/>
        <param name="image_name_right"  type="string" value="right_"/>
        <param name="board_width"       type="int"  value="9"/>
        <param name="board_height"      type="int"  value="8"/>
        <param name="is_use_OpenCV"     type="bool" value="false"/>
        <param name="is_show"           type="bool" value="true"/>
    </node>
</launch>

